sdk/logs/src/main/java/io/opentelemetry/sdk/logs/internal/SdkEventEmitterProvider.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.sdk.logs;
+package io.opentelemetry.sdk.logs.internal;
```

Comment:
In #5341 I propose we move `SdkEventEmitterProvder` to an internal package rather than splitting out into a separate module. We should make the move before the release so the only change in the next release is marking the module as stable.